docs/features/software-catalog/system-model.md

```diff
@@ -3,55 +3,24 @@ id: system-model
 title: System Model
 ---
 
-We believe that a strong shared understanding and terminology around systems,
-software and resources leads to a better Backstage experience.
+We believe that a strong shared understanding and terminology around software
+and resources leads to a better Backstage experience.
 
 _This description originates from
 [this RFC](https://github.com/spotify/backstage/issues/390). Note that some of
 the concepts are not yet supported in Backstage._
 
-## Concepts
+## Core Entities
 
-We model our technology using these five concepts (further explained below):
+We model software in the Backstage catalogue using these three core entities
+(further explained below):
 
-- **Domains** are a high-level grouping of systems
```

Comment:
Till now I expected Domains and Systems to be entities, too? Does that change, or will it be added again once they are implemented?

Reply:
We had a lot of internal discussion around not enforcing the system concept for all components. Shared libraries or modules in a mobile application might not have a system to attach them to (or sometimes, several ones). It'd still be nice if Backstage would have some support for systems though, even if associating entities with a system is optional. If you think that is valuable, we can keep it around?

Reply:
Till now they looked quite optional, but in bigger platform I would expect them to be essential. 

I don't think that having the concept would enforce it for the users. I know Systems and Domain don't exist yet, but it is something we would be happy to contribute once the basics of #1964 are done.

Reply:
Alright, I added them back. What do you think?

Reply:
Looks good, later when they are implemented in backstage we can add another picture to `Ecosystem Modeling` how these are part of the model.

Reply:
Yeah, for now I wanted to leave us some room to figure out how exactly to do it.